pgactivity/ui.py

```diff
@@ -229,11 +237,9 @@ def main(
                     term,
                     ui,
                     host=host,
-                    dbinfo=dbinfo,
                     pg_version=data.pg_version,
-                    tps=tps,
-                    active_connections=active_connections,
-                    activity_stats=activity_stats,
+                    server_information=server_information,
+                    activity_stats=activity_stats,  # FIXME
```

Comment:
What's the `FIXME` for?

Reply:
I wasn't super fond of the tuple we use there.
But I don't have anything better to propose.

Removed !